folly/Portability.h

```diff
@@ -219,7 +221,9 @@ FOLLY_GCC_DISABLE_NEW_SHADOW_WARNINGS
 // the 'std' namespace; the latter uses inline namespaces. Wrap this decision
 // up in a macro to make forward-declarations easier.
 #if FOLLY_USE_LIBCPP
+#if __has_include(<__config>)
```

Comment:
This is already guarded. Why is a second guard necessary?

`FOLLY_USE_LIBCPP` should not be set if cpplib is in use. If it is, the bug is there.

Reply:
probably an artifact from when I was fighting a ghost folly-config.h somewhere in my include path. fixed.